utils/crates-io/src/manifest.rs

```diff
@@ -46,23 +48,141 @@ impl ManifestWithPath {
             .canonicalize()?;
 
         Ok(Self {
-            name: "__gear_workspace".into(),
-            manifest: Manifest::from_path(&path)?,
+            name: WORKSPACE_NAME.to_string(),
+            manifest: fs::read_to_string(&path)?.parse()?,
             path,
         })
     }
 
     /// Complete the manifest of the specified crate from
-    /// the current manifest
-    pub fn manifest(&self, path: impl AsRef<Path>) -> Result<Self> {
-        let mut manifest = Manifest::<Value>::from_slice_with_metadata(&fs::read(&path)?)?;
-        manifest
-            .complete_from_path_and_workspace(path.as_ref(), Some((&self.manifest, &self.path)))?;
+    /// the workspace manifest
+    pub fn manifest(&self, pkg: &Package) -> Result<Self> {
+        self.ensure_workspace()?;
+
+        // Complete documentation as from <https://docs.rs>
+        let mut manifest: Document = fs::read_to_string(&pkg.manifest_path)?.parse()?;
+        let name = pkg.name.clone();
+        manifest["package"]["documentation"] = toml_edit::value(format!("https://docs.rs/{name}"));
```

Comment:
since `toml_edit` supports `serde` feature, maybe we can declare some `struct` with the contents of `Cargo.toml` and modify it instead of accesing keys?

Reply:
3f03d09

Reply:
iirc serde has flatten attr for that

Reply:
https://github.com/gear-tech/gear/issues/3583